config/routes.rb

```diff
@@ -2,14 +2,15 @@
 
 
 
- root 'static_pages#home'
-  get '/signup', to: 'users#new'
-  post '/signup', to: 'users#create'
-  get '/login', to: 'sessions#new'
-  post '/login', to: 'sessions#create'
-  delete '/logout', to: 'sessions#destroy'
-  get '/new', to: 'posts#new'
-  post '/create', to: 'posts#create'
+  root "static_pages#home"
+  get "/signup", to: "users#new"
+  post "/signup", to: "users#create"
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
+  get "/new", to: "posts#new"
+  post "/create", to: "posts#create"
```

Comment:
Why you need to use line 11, 12?

Reply:
bỏ đi chị ạ. Bên dưới có resources :posts rồi :))